src/background/index.ts

```diff
@@ -3,3 +3,8 @@ import { storage } from "../storage";
 chrome.runtime.onInstalled.addListener(() => {
     storage.get().then(console.log);
 });
+
+// Allows users to open the side panel by clicking on the action toolbar icon
+chrome.sidePanel.setPanelBehavior({ openPanelOnActionClick: true })
```

Comment:
Unfortunately this code conflicts with `src/popup/popup.html` which also opens by clicking on the action toolbar icon :(

<img width="286" alt="image" src="https://github.com/NekitCorp/chrome-extension-svelte-typescript-boilerplate/assets/15874982/d61223ec-aa22-4ed2-a3df-8c540a19dabe">

Reply:
Updated to offer the side panel variation as an option, but commented out.

Reply:
Great! Thanks you!